openedxstats/apps/sites/tests.py

```diff
@@ -375,3 +379,33 @@ def test_site_get_geographies_method_with_unicode(self):
         self.assertEqual(str(sitegeozone1), "https://www.κόσμε.co---Greece")
         self.assertEqual(str(geozone2), "\u00e9")
 
+
+class OTChartTestCase(TestCase):
+    def setUp(self):
+        user = User.objects.create_user('testuser', 'testuser@edx.com', 'password')
+        if user is None:
+            self.fail("Could not create testuser in setUp()")
+        self.client.login(username='testuser', password='password')
+
+    def test_nothing_imported_from_ajax_call(self):
+        # Ajax request
+        response = self.client.post('/sites/ot_chart/', context_type='application/json',
+                                    HTTP_X_REQUESTED_WITH='XMLHttpRequest')
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content.decode(), json.dumps([]))
+
+    def test_json_data_returned_after_ajax_call(self):
+        snapshot = SiteSummarySnapshot(
+            timestamp=datetime(2016, 7, 1, 0, 0, 0),
+            num_sites=100,
+            num_courses=1000,
+            notes="test"
+        )
+        snapshot.save()
+        # Ajax request
+        response = self.client.post('/sites/ot_chart/', context_type='application/json',
+                                    HTTP_X_REQUESTED_WITH='XMLHttpRequest')
+        self.assertEqual(response.status_code, 200)
+        expected_json = json.loads(serialize('json', [snapshot]))
+        response_json = json.loads(response.content.decode())
+        self.assertEqual(expected_json[0], response_json[0])
```

Comment:
The other elements should be empty because there is only one data point, I've corrected it to check to see if they're empty